### PR TITLE
Japanese translation improvement

### PR DESCRIPTION
**Pull request by yuhattor**
- Fixed and Unified some expressions on Trusted Committer.
- Fixed markdown 
#434

### CODE REVIEW

**Comment by yuhattor** [comment]
@spier Review is required, Just a quick review please:)

File: translation/japanese/patterns/trusted-committer.md
Comment (spier) [inline translation/japanese/patterns/trusted-committer.md:3]: @yuhattor This looks like you had multiple translations for TC here?

The main one, and then a second one in brackets?

Note that you can use the '## Alias' section for such alternative titles.

**Review by spier — APPROVED** [review]
I can not review the Japanese itself but when you are happy with it, so am I ;)

Maybe for future PR could we get a second person from the JP patterns community to review these PRs from a pure language perspective?

Lastly, before merging this PR, please confirm if the failing link check iss as in one of your files.

**Comment by spier** [comment]
@yuhattor please change the PR target to 'main'. All changes need to be made there first and then pushed to 'book-jp' later. 

I am not at my laptop so cannot makes these changes right now. Also excuse my brevity. 

I can help tomorrow to get these changes published in the book.

**Comment by yuhattor** [comment]
@spier 
> I can not review the Japanese itself but when you are happy with it, so am I ;)
> Maybe for future PR could we get a second person from the JP patterns community to review these PRs from a pure language perspective?

Thank you:) Yea.. Our community in Japan is still small, but I hope we will have more contributers soon!

> Lastly, before merging this PR, please confirm if the failing link check iss as in one of your files.
Sure, it looks the error is caused by GitHub Actions sending too many requests, resulting in 429.

I wonder why it failed only this time, looking at the lycheeverse/lychee settings, it looks like the default values regarding retries are applied

```
  ERROR	https://github.com/tapjdey/InnerSource_Project_Fitness
    Too Many Requests (HTTP error 429)
```

Even if I re-run it a few times, it gets rejected due to strict throttling on the GitHub side...

image

**Comment by yuhattor** [comment]
>  please change the PR target to 'main'. All changes need to be made there first and then pushed to 'book-jp' later.
I am not at my laptop so cannot makes these changes right now. Also excuse my brevity.
I can help tomorrow to get these changes published in the book.

Oh, that's right. Thanks for pointing that out!

File: translation/japanese/patterns/trusted-committer.md
Comment (yuhattor) [reply translation/japanese/patterns/trusted-committer.md:3]: TC is a very important word, and we had a deep discussion in the community on how to express it.
Long story short, we decided not to translate "Trusted Committer" as suggested by ICS. Japanese has katakana, a character that represents a sound.

So we are okay with using one word here, but if we need to alias it in the future for other terms, we will do so

**Comment by yuhattor** [comment]
Regarding the rate limit, it looks like I found a way to avoid it.

> GitHub Token
To avoid getting rate-limited while checking GitHub links, you can optionally set an environment variable with your Github token like so GITHUB_TOKEN=xxxx, or use the --github-token CLI option. It can also be set in the config file. [Here is an example config file](https://github.com/lycheeverse/lychee/blob/master/lychee.example.toml).

https://github.com/lycheeverse/lychee

**Comment by spier** [comment]
I believe we are already using they. 
See `.github/workflows/link-checker.yml`

After adding that the GitHub rate limiting issues were resolved. However apparently not for good. 

That token is generated by GHA on every CI run. 
I suspect it has the same rate limits as a personal access token would have. Not sure though. 

I say let's merge this PR into main, and watch how the link checker behaves in the future. 



> Regarding the rate limit, it looks like I found a way to avoid it.
> 
> 
> 
> > GitHub Token
> 
> To avoid getting rate-limited while checking GitHub links, you can optionally set an environment variable with your Github token like so GITHUB_TOKEN=xxxx, or use the --github-token CLI option. It can also be set in the config file. [Here is an example config file](https://github.com/lycheeverse/lychee/blob/master/lychee.example.toml).
> 
> 
> 
> https://github.com/lycheeverse/lychee
> 
>

File: translation/japanese/patterns/trusted-committer.md
Comment (spier) [reply translation/japanese/patterns/trusted-committer.md:3]: One day you need to tell me more about what "katakana" means. Sounds great for sure :)

But for now, let's resolve this thread.

**Review by spier — APPROVED** [review]

**Comment by spier** [comment]
@yuhattor I got access to a proper laptop again.

Changed the base branch of this PR to `main`, and approved the PR.

If this all looks good to you, please merge.

I can then help tomorrow to get this released (i.e. pushed from `main` to `book-jp`.

**Comment by yuhattor** [comment]
Thank you for coordinating pull requests!!
Also, thanks for the comment about rate limits as well.
Yes, let's merge them once:)

**Pull request merged**
